data/last.json

```diff
@@ -99,5 +99,6 @@
     "Butler",
     "Barnes",
     "Fisher",
-    "Bale"
+    "Bale",
+    "Trump"
```

Comment:
I think this should actually be spelt "Drumpf", I'd be happy to help you with that.